packages/nexus-sdk/src/nexusSdk.ts

```diff
@@ -36,7 +38,21 @@ export type NexusContext = Context & {
 };
 
 export function createNexusClient(options: NexusClientOptions) {
-  const defaultLinks = [triggerFetch(options.fetch)];
+  // get fetch instance
+  const fetchInstance = options.fetch || getFetchInstance();
+  if (!fetchInstance) {
+    throw new Error(
+      'No `fetch` instance was found. It could be because your browser requires a polyfill or maybe you are trying to run this in Node.js in which case you should use node-fetch or equivalent. You can find more information here: https://github.com/BlueBrain/nexus-js/tree/master/packages/nexus-sdk#nodejs-support',
```

Comment:
I think it would be nice if you put some commas, so it is easier to read, like:

No `fetch` instance was found. It could be because your browser requires a polyfill, or maybe you are trying to run this in Node.js, in which case you should use node-fetch or equivalent.

Reply:
ah yes, the oxford comma